ApplyToBecomeInternal/DocumentGeneration/Builders/DocumentBodyBuilder.cs

```diff
@@ -10,14 +10,12 @@ namespace DocumentGeneration.Builders
 {
 	public class DocumentBodyBuilder : IDocumentBodyBuilder
 	{
-		private readonly WordprocessingDocument _document;
-		private readonly Body _body;
+		private readonly WordprocessingDocument _document;		
 		private OpenXmlElement _previousElement;
 
 		public DocumentBodyBuilder(WordprocessingDocument document, OpenXmlElement previousElement)
 		{
 			_document = document;
-			_body = document.MainDocumentPart.Document.Body;

```

Comment:
Happy to approve, assuming the generated doc is still working without the body property?

Reply:
Good point. I'll double check, saw it wasn't being used so assumed it could be removed.

Reply:
Have checked and doc generation works fine.

Reply:
All good, just one code smell now then happy to approve.

Reply:
Thanks @dneed-nimble . This has now been addressed.